### PR TITLE
fix: update livebook and README.md under sdk/elixir

### PR DESCRIPTION
**Pull request by wingyplus**
Update livebook file to make it compatible with current SDK.

### CODE REVIEW

File: sdk/elixir/getting_started.livemd [hunk 1]
```diff
@@ -41,10 +41,10 @@ In this section, we will pull Elixir image from [hex.pm](https://hub.docker.com/
 After execute the code below, the library will printing out log from Dagger session to the standard output.
 
 ```elixir
-client
-|> Dagger.Query.container()
-|> Dagger.Container.from("hexpm/elixir:1.14.4-erlang-25.3-debian-buster-20230227-slim")
-|> Dagger.Container.with_exec(["elixir", "--version"])
-|> Dagger.Container.stdout()
-|> IO.puts()
```
Comment (gmile) [inline sdk/elixir/getting_started.livemd:49]: Is removing `IO.puts` mean nothing will be printed out in the terminal? 🤔

Comment (wingyplus) [reply sdk/elixir/getting_started.livemd:49]: By default `stdout` will print through dagger log.  So I think it is not necessary to add `IO.puts` in this cell. :)

Comment (wingyplus) [reply sdk/elixir/getting_started.livemd:49]: Here is demo 
Screenshot 2566-08-04 at 21 03 30

Comment (gmile) [reply sdk/elixir/getting_started.livemd:49]: @wingyplus ah, that's actually nice, I didn't know that!

**Review by gerhard — APPROVED** [review]
Merge when ready @wingyplus 💪

**Pull request merged**